x-pack/plugins/cases/public/containers/use_get_case_user_actions_stats.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { getCaseUserActionsStats } from './api';
+
+import type { ServerError } from '../types';
+import { useToasts } from '../common/lib/kibana';
+import { ERROR_TITLE } from './translations';
+import { casesQueriesKeys } from './constants';
+
+export const useGetCaseUserActionsStats = (caseId: string) => {
+  const toasts = useToasts();
```

Comment:
To avoid code duplication there is a hook called `useCasesToast` which you can use. You can see `x-pack/plugins/cases/public/containers/use_delete_cases.tsx` as an example.